app/views/lists/index.html.erb

```diff
@@ -1 +1,24 @@
-<h1 class="title">Listr Index</h1>
\ No newline at end of file
+<h1 class="title">Listr Index</h1>
```

Comment:
Listr?

Reply:
Was thinking about naming it that, but changed to List. Resolved afterwards in latest branch.